tools/pd-ctl/tests/config/config_test.go

```diff
@@ -545,7 +545,10 @@ func (suite *configTestSuite) checkConfigForwardControl(cluster *pdTests.TestClu
 }
 
 func (suite *configTestSuite) TestPlacementRules() {
-	suite.env.RunTestInTwoModes(suite.checkPlacementRules)
+	// use a new environment to avoid affecting other tests
+	env := pdTests.NewSchedulingTestEnvironment(suite.T())
+	env.RunTestInTwoModes(suite.checkPlacementRules)
+	env.Cleanup()
```

Comment:
What about moving `NewSchedulingTestEnvironment` into `SetupTest` from `SetupSuite` to make sure each test uses a different env?

Reply:
I think setup just only once in a suite 0 0

Reply:
`SetupSuite` only runs once in the same suite, while `SetupTest` runs each time before a test function in a suite.

https://github.com/stretchr/testify/blob/v1.8.4/suite/interfaces.go#L21

Reply:
get it